### DISCUSSION TITLE
selected and postOnEsc for enterFindMode

### DISCUSSION THREAD

**Discussion by ghost** [gdh1995/vimium-c#1050] [Q&A] [answered]
I'm curious about the usage of these two commands, haven't notice anything differences after remove these two and hardly find hint in the source code.

**Comment by gdh1995** [comment, marked as answer]
The parameter of `selected` (aka. `selected=true`) means `FindMode` should auto use selected text to execute a query at once.
- This parameter is enabled by default, and you may disable this behavior by appending ` selected=false` into `map xxx enterFindMode`

`postOnEsc` is also a boolean (true/false) parameter while it's disabled by default. It means to execute a `postActivate` action on FindMode receiving an `<esc>` key.
But, during my test on Chrome 119, this seems to fail as you said - maybe you can forget it...
